namui/src/namui/mod.rs

```diff
@@ -61,7 +61,10 @@ pub async fn start<TProps>(
     let mut event_count = 0;
 
     loop {
-        let event = event_receiver.recv().await.unwrap();
+        let event = event_receiver
```

Comment:
Please run cargo fmt in namui too. I think this pr should not add commit about namui.

Reply:
Fixed with c088d01